README.md

```diff
@@ -1,158 +1,111 @@
 
-#### endpoint-secret.yaml
+1.  `kubevirt.io/storage.import.secretName:` (Optional) The name of the secret containing the authentication credentials required by the file server
 
-One or more endpoint secrets in the "golden" namespace are required for non-public endpoints. If the endpoint is public there is no need to an endpoint secret. No namespace is supplied since the secret is expected to be created from the "golden" namespace.
+###### Edit endpoint-secret.yaml:
 
-##### Edit:
-- `metadata.name:` change this to a different secert name if desired. Remember to use this name in the PVC's secret annotation.
--  `accessKeyId:` to contain the endpoint's key and/or user name. This value must be **base64** encoded with no extraneous linefeeds. Use `echo -n "xyzzy" | base64` or `printf "xyzzy" | base64` to avoid a trailing linefeed.
--  `secretKey:`  the endpoint's secret or password, again base64 encoded.
-The credentials provided here are consumed by the S3 client inside the pod.
-> NOTE: the access key id and secret key **must** be base64 encoded without newlines (\n).
+> Note:  (Optional) Only set these values if the file server requires authentication credentials.
 
-#### golden-pvc.yaml
+1. `metadata.name:` Arbitrary name of the secret. Must match the PVC's `kubevirt.io/storage.import.secretName:`
 
-This is the template PVC. No namespace is supplied since the PVC is excpected to be created from the "golden" namespace. A storage class is also not provided since there is excpected to be a default storage class per cluster. A storage class will need to be added if the default storage provider does not met the needs of golden images. For example, when copying VM image files, the backend storage should support fast-cloning, and thus a non-default storage class may be needed.
+1.  `accessKeyId:` Contains the endpoint's key and/or user name. This value **must be base64 encoded** with no extraneous linefeeds. Use `echo -n "xyzzy" | base64` or `printf "xyzzy" | base64` to avoid a trailing linefeed
 
-##### Edit:
--  `storageClassName:` change this to the desired storage class for high speed cloning.
--  `kubevirt.io/storage.import.endpoint:` change this to contain the source endpoint. Format: `(http||s3)://www.myUrl.com/path/of/data`
--  `kubevirt.io/storage.import.secretName:` (not needed for public endpoints). Edit the name of the secret containing credentials for the supplied endpoint.
+1.  `secretKey:`  the endpoint's secret or password, again **base64 encoded**
 
 ### Deploy the API Objects
 
-1. First, create the "golden" namespace:  (no manifests are provided)
+1. (Optional) Create the "golden" namespace
 
-1. Next, create one or more storage classes: (no manifests are provided).
+    `$ kubectl create ns <name>`
 
-1. Next, create the endpoint secrets:
+1. (Optional) Create the endpoint secrets:
 
-        $ kubectl create -f endpoint-secret.yaml
+   `$ kubectl create -f endpoint-secret.yaml`
```

Comment:
mention that all of these _create -f_s are done in the target ("Golden") namespace.